physionet-django/project/modelcomponents/metadata.py

```diff
@@ -1,5 +1,6 @@
 import os
 
+from django.conf import settings
```

Comment:
Is this being used? I don't see it in this module. I guess it may be used in a downstream import. If so, should we add a comment to help prevent it from being removed, or instead make the import wherever it is used downstream?